### PR TITLE
Add an sshkey for a gerrit user

### PR DESCRIPTION
**Pull request by bogdando**
Signed-off-by: Bogdan Dobrelya bdobrelia@mirantis.com

### CODE REVIEW

File: make-gerrit-key.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+# Env vars expected:
+# SECRET_FILE, ADMIN_USER, USERNAME
+
+. config
+
+SECRET_FILE="${SECRET_FILE:-~/keys/jenkins_mcp.pub}"
+KEY="$(cat ${SECRET_FILE})"
+$ADMIN_USER="${ADMIN_USER:-admin}"
+$USERNAME="${USERNAME:-mcptestuser}"
```
Comment (mattymo) [inline make-gerrit-key.sh]: Please call this target_user

File: make-gerrit-key.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+# Env vars expected:
+# SECRET_FILE, ADMIN_USER, USERNAME
+
+. config
+
+SECRET_FILE="${SECRET_FILE:-~/keys/jenkins_mcp.pub}"
+KEY="$(cat ${SECRET_FILE})"
+$ADMIN_USER="${ADMIN_USER:-admin}"
```
Comment (mattymo) [inline make-gerrit-key.sh]: Please call this gerrit_user

File: make-gerrit-key.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+$USERNAME="${USERNAME:-mcptestuser}"
+
+ssh  -p 29418 ${ADMIN_USER}@review.fuel-infra.org gerrit set-account \
```
Comment (mattymo) [inline make-gerrit-key.sh]: Please make port and gerrit_uri configurable

File: make-gerrit-key.sh (hunk 2 above)
Comment (bogdando) [reply make-gerrit-key.sh]: But this is the admin gerrit user. The gerrit user is a USERNAME

**Pull request merged**